src/styles/main.scss

```diff
@@ -1,3 +1,16 @@
+@import "./utils/variables";
+@import "./utils/functions";
+
+@import "./blocks/calendar.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
It's not the best idea to add a display: flex to the body tag. Would be better to use class name here